iml-graphql-queries/src/snapshot.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+pub mod create {
+    use crate::Query;
+
+    pub const QUERY: &str = r#"
+            mutation CreateSnapshot($fsname: String!, $name: String!, $comment: String, $use_barrier: Boolean) {
+              createSnapshot(fsname: $fsname, name: $name, comment: $comment, use_barrier: $use_barrier) {
+                cancelled
+                complete
+                createdAt
```

Comment:
```suggestion
                created_at: createdAt
```
For smooth transition.

Reply:
should we alias all of the fields then? (probably)

Reply:
nm.

Reply:
nice fix.